rotkehlchen/accounting/constants.py

```diff
@@ -33,6 +33,8 @@
         HistoryEventSubType.DONATE: EventCategory.RECEIVE_DONATION,
         HistoryEventSubType.NONE: EventCategory.RECEIVE,
         HistoryEventSubType.LIQUIDATE: EventCategory.LIQUIDATION_REWARD,
+        HistoryEventSubType.PAYMENT: EventCategory.RECEIVE_PAYMENT,
+        HistoryEventSubType.GRANT: EventCategory.RECEIVE_GRANT,
```

Comment:
where are those 2 defined? I don't see them in bugfixes

Reply:
In this PR. But one mapping is missing which I will push.